pydis_site/apps/content/resources/guides/pydis-guides/help-channel-guide.md

```diff
@@ -10,66 +10,84 @@ relevant_links:
 
-Some of the topical help channels have a broad scope, so they can cover many (somewhat) related topics.
+Some of the topical help channels have a broad scope, so they can cover many related topics.
 For example, `#data-science-and-ai` covers scientific Python, statistics, and machine learning, while `#algos-and-data-structs` covers everything from data structures and algorithms to maths.
 
-To help you navigate this, we've added a list of suggested topics in the topic of every channel.
-If you're not sure where to post, feel free to ask us which channel is relevant for a topic in `#community-meta`.
+Each channel on the server has a channel description which briefly describes the topics covered by that channel. If you're not sure where to post, feel free to ask us which channel is appropriate in `#community-meta`.
 
 # General Help Channels
 
-Our general help channels move at a fast pace, and attract a far more diverse spectrum of helpers.
-This is a great choice for a generic Python question, and a good choice if you need an answer as soon as possible.
-It's particularly important to [ask good questions](../asking-good-questions) when asking in these channels, or you risk not getting an answer and having your help channel be claimed by someone else.
+Our general help channels cycle quickly, with the advantage of attracting a more diverse spectrum of helpers, and getting individual focus and attention on your question. These channels are a great choice for generic Python help, but can be used for domain-specific Python help as well.
 
-## How To Claim a Channel
+## How to Claim a Channel
 
-There are always 3 available help channels waiting to be claimed in the **Python Help: Available** category.
+There are always three help channels waiting to be claimed in the **Available Help Channels** category.
 
 ![Available help channels](/static/images/content/help_channels/available_channels.png)
+*The Available Help Channels category is always at the top of the server's channel list.*
 
-In order to claim one, simply start typing your question into one of these channels. Once your question has been posted, you have claimed this channel, and the channel will be moved down to the **Python Help: Occupied** category.
+![Available message](/static/images/content/help_channels/available_message.png)
+*This message indicates that a channel is available.*
 
-If you're unable to type into these channels, this means you're currently **on cooldown**. In order to prevent someone from claiming all the channels for themselves, **we only allow someone to claim a new help channel every 15 minutes**. However, if you close your help channel using the `!dormant` command, this cooldown is reset early.
+In order to claim one, simply ask your question in one of the available channels. Be sure to [ask good questions](../asking-good-questions) in order to give yourself the best chances of getting help!
```

Comment:
maybe we could call it "asking questions well" or something? someone might think that their problem might not be good enough to be asked about or something.

Reply:
Rephrasing to "Be sure to ask questions with enough information..."